content/docs/1.6.0/advanced-resources/deploy/node-selector.md

```diff
@@ -43,20 +43,20 @@ You need to set node selector for both types of components. See more details bel
    >   systemManagedComponentsNodeSelector: "label-key1:label-value1"
    >  ```
 
-
 ### Setting up Node Selector After Longhorn has been installed
 
 > **Warning**:
 > * Since all Longhorn components will be restarted, the Longhorn system is unavailable temporarily.
-> * Make sure all Longhorn volumes are `detached`. If there are running Longhorn volumes in the system, this means the Longhorn system cannot restart its components and the request will be rejected.
+> * When all Longhorn volumes are detached, the customized setting is immediately applied to the system-managed components.
```

Comment:
Can you briefly describe what components will be applied with the setting.

Reply:
Added what the components will be applied with the setting.